api/src/main/resources/workbench.yaml

```diff
@@ -583,88 +598,77 @@ paths:
-          required: true
-          type: string
+        - $ref: '#/parameters/workspaceNamespace'
       responses:
         200:
-          description: The users cluster
+          description: The cluster for this user and workspace.
           schema:
-            $ref: '#/definitions/DefaultClusterResponse'
-        500:
-          description: Internal Error
+            $ref: '#/definitions/Cluster'
+        404:
+          description: No cluster exists for this user and workspace.
           schema:
             $ref: '#/definitions/ErrorResponse'
-
-  /v1/clusters/{clusterNamespace}/{clusterName}:
+    post:
+      summary: Create a workspace cluster.
+      description: >
+        Creates a new cluster for the current user in the given billing project. If a cluster already
+        exists for the user in this billing project, a 409 conflict error is returned (even if the cluster
+        is still initializing or is not in a ready state).
+      operationId: createCluster
+      tags:
+        - cluster
+      parameters:
+        # TODO(RW-3697): Custom cluster creation params should be added as a body param here.
+        - $ref: '#/parameters/workspaceNamespace'
+      responses:
+        200:
+          description: Returns the cluster that was created for this user and workspace.
+          schema:
+            $ref: '#/definitions/Cluster'
+        409:
+          description: A cluster for this user and workspace already exists.
+          schema:
+            $ref: "#/definitions/EmptyResponse"
+    # TODO(RW-3695): updateCluster should be added as a PATCH endpoint here.
     delete:
-      summary: Delete a cluster by name.
+      summary: Delete a workspace cluster.
       operationId: deleteCluster
       tags:
         - cluster
       parameters:
-        - in: path
-          name: clusterNamespace
-          description: clusterNamespace
-          required: true
-          type: string
-        - in: path
-          name: clusterName
-          description: clusterName
-          required: true
-          type: string
+        - $ref: '#/parameters/workspaceNamespace'
       responses:
         200:
-          description: Deletion success
+          description: Success
           schema:
             $ref: '#/definitions/EmptyResponse'
-        500:
-          description: Internal Error
+        404:
+          description: No cluster exists for this user and workspace.
           schema:
-            $ref: '#/definitions/ErrorResponse'
+            $ref: "#/definitions/ErrorResponse"
 
-  /v1/clusters/{clusterNamespace}/{clusterName}/localize:
+  /v1/clusters/{workspaceNamespace}/localize:
     post:
-      summary: >
-        Localize files from a workspace to notebook cluster. As a side-effect,
+      summary: Localize files to the user's cluster.
+      description: >
+        Localizes files to the cluster for the current user and given workspace. As a side-effect,
```

Comment:
I guess I'm still confused on when this should be called in the lifecycle of the cluster. Is it during startup (i.e. is this the thing that gives users their files back?)  Maybe also replace "localizes" with "copies", unless there's some other translation happening.